### DISCUSSION TITLE
http/https-summary

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#209] [General]
# http/https-summary

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/http/https-summary.html

**Comment by tessEcho** [comment]
它定义一系列关网络安全概念和组件 -> 它定义了一系列网络安全概念和组件

**Reply by isno** [reply]
谢谢，已修改。

**Comment by zjyyzhengli** [comment]
奇怪，左侧导航栏中的标题名称（2.5 HTTPS加密原理与优化实践）和页面中的标题名称（2.5 HTTPS原理及实践）不一致。
